spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -452,6 +456,22 @@ private void declareQueue(String beanName, Queue queue) {
 		addToAutoDeclareContext(beanName, queue);
 	}
 
+	private Exchange buildExchange(RabbitCommonProperties properties, String exchangeName) {
+		try {
+			// TODO Make the ctor public in Spring-AMQP - AMQP-695
+			Constructor<ExchangeBuilder> ctor = ExchangeBuilder.class.getDeclaredConstructor(String.class, String.class);
```

Comment:
We've just released 1.2.0.M1 so I suppose it could wait a bit. On the other hand, we've never had good experiences with PRs sitting for a long while, so perhaps we should merge as soon as possible.